### ISSUE TITLE
reg loss  can not fall 

### ISSUE THREAD

**Issue by leiqing110** [zylo117/Yet-Another-EfficientDet-Pytorch#593]
Hello author, I trained VisDrone dataset and loss can't fall. Can you take a look at it for me, thank you
![1111](https://user-images.githubusercontent.com/19700402/106694150-f32a1d00-6612-11eb-8a3f-86545559c920.png)

train：python train.py -c 0 -p visdrone2019 --batch_size 8 --lr 1e-5 --num_epochs 50

**Comment by leiqing110** [comment]
我把数据集上传到谷歌网盘了https://drive.google.com/file/d/1O7ic0W9xhCqE_KXV06iNTuP0h970sF69/view?usp=sharing

**Comment by zylo117** [comment]
try lr 1e-3

**Comment by leiqing110** [comment]
> try lr 1e-3

Thank you for your reply. After I retrained, the loss could go down. But the final ap is only 0.03. I don't know why
![Snipaste_2021-02-06_09-56-34](https://user-images.githubusercontent.com/19700402/107105697-8f962e80-6862-11eb-9724-fbdf485fdc30.png)

**Comment by leiqing110** [comment]
> try lr 1e-3

![Snipaste_2021-02-06_13-52-02](https://user-images.githubusercontent.com/19700402/107110426-9b91e880-6882-11eb-878b-d5b8a8dbab1a.png)

**Comment by lihao4096** [comment]
> > try lr 1e-3
> 
> ![Snipaste_2021-02-06_13-52-02](https://user-images.githubusercontent.com/19700402/107110426-9b91e880-6882-11eb-878b-d5b8a8dbab1a.png)

same question. Have you find the solution？
